types/ext/language.d.ts

```diff
@@ -51,4 +51,5 @@ export type LanguageSummary = {
     name: string;
     iso: string;
     exampleText: string;
+    languageTransformsFile?: string;
```

Comment:
Thoughts on making this the full file name and potentially including the path? Right now it seems unintuitive that the JSON is missing, which can make CTRL+F in the codebase more difficult to find usages of things (e.g. if I searched for `english-transforms.json` it wouldn't come up).

Path I suppose is more arguable either way.